src/routing/ActiveLink.ts

```diff
@@ -14,6 +14,10 @@ export interface ActiveLinkProperties extends LinkProperties {
 export class ActiveLink extends WidgetBase<ActiveLinkProperties> {
 	private _outletHandle: Handle | undefined;
 
+	private _paramsEqual(linkParams: any = {}, contextParams: any = {}) {
```

Comment:
This doesn't actually have to be on the `ActiveLink` instance

Reply:
Good point! I guess we can move this out into a fn.